docs/install/azure.rst

```diff
@@ -13,16 +13,16 @@ users and a user environment with packages you want to be installed running on
 
 This tutorial leads you step-by-step for you to manually deploy your own JupyterHub on Azure cloud.
 
-.. note:: ✨ The ``Deploy to Azure button`` project allows you  to deploy your own JupyterHub with minimal manual configuration steps. The deploy to Azure button allows you  to have a vanilla configuration in just one-click and by assigning some variables. 
-    
+.. note:: ✨ The ``Deploy to Azure button`` project allows you  to deploy your own JupyterHub with minimal manual configuration steps. The deploy to Azure button allows you  to have a vanilla configuration in just one-click and by assigning some variables.
+
     Check it out at `https://github.com/trallard/TLJH-azure-button <https://github.com/trallard/TLJH-azure-button>`_.
 
 Prerequisites
 ==============
 
-* A Microsoft Azure account. 
+* A Microsoft Azure account.
 
-* To get started you can get a free account which  includes 150 dollars worth of Azure credits (`get a free account here <https://azure.microsoft.com/en-us/free//?wt.mc_id=TLJH-github-taallard>`_) 
+* To get started you can get a free account which  includes 150 dollars worth of Azure credits (`get a free account here <https://azure.microsoft.com/en-us/free//?wt.mc_id=TLJH-github-taallard>`_)
```

Comment:
Do we need the tracking(?) identifier here and in the other Azure links?

Reply:
I'm not sure, @trallard can you comment on the campaign ID in the azure links (added in #342)?

Reply:
I opened https://github.com/jupyterhub/the-littlest-jupyterhub/issues/860 to track this so that we can move onwards with this PR